src/app/dataset-view/additional-components/dataset-settings-component/tabs/compacting/dataset-settings-compacting-tab.types.ts

```diff
@@ -9,3 +9,13 @@ export const sliceSizeMapper: Record<SliceUnit, number> = {
     [SliceUnit.MB]: Math.pow(2, 20),
     [SliceUnit.GB]: Math.pow(2, 30),
 };
+
+export function sliceSizeMapperReverse(sizeInBytes: number): { size: number; unit: SliceUnit } {
+    if (sizeInBytes % Math.pow(2, 30) === 0) {
+        return { size: sizeInBytes / Math.pow(2, 30), unit: SliceUnit.GB };
+    } else if (sizeInBytes % Math.pow(2, 20) === 0) {
+        return { size: sizeInBytes / Math.pow(2, 20), unit: SliceUnit.MB };
+    } else {
+        return { size: sizeInBytes / Math.pow(2, 10), unit: SliceUnit.KB };
+    }
+}
```

Comment:
I think this isn't specific to the component, and maybe moved somewhere to common utility code

Reply:
Moved to app helpers